app/controllers/application_controller.rb

```diff
@@ -16,11 +17,14 @@ def after_sign_in_path_for(user)
     end
   end
 
+  def set_language
+    @language = 1 || Language.find_by(id: 1).id
```

Comment:
@coreyd303 Is this supposed to be an assignment? ```@language = 1``` should assign @language to be 1, which will always return truthy, so the later half of the conditional will never run.

I suspect you want to check ```@language.id == 1```, not assign it.